index.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module "boring-avatars" {
+  interface AvatarProps {
```

Comment:
```suggestion
  export interface AvatarProps {
```

Exporting these makes it simpler to build custom components that wrap the Avatar components. Wdyt?